src/main/java/core/basesyntax/report/output/Writer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.report.output;
+
+public interface Writer {
+    public void write(String resultingReport, String pathFile);
```

Comment:
You should omit the 'public' modifier in interface method declarations as methods in an interface are public by default.